chcemvediet/apps/inforequests/admin.py

```diff
@@ -397,3 +401,11 @@ def render_delete_form(self, request, context):
                 may cause logical errors in the inforequest history.
                 """)))
         return super(ActionAdmin, self).render_delete_form(request, context)
+
+    def delete_model(self, request, obj):
+        if request.POST:
+            if request.POST.get(u'snooze'):
+                previous = obj.previous_action
+                previous.snooze = local_today() + datetime.timedelta(days=7)
```

Comment:
@martinmacko47 Mozem tomu nechat pevne cislo, alebo to radsej prepisat cez konstantu. Islo o to ze aj inde v kode boli niekde "natvrdo" zadane cisla.

Reply:
Asi by bolo fajn, aby to bola pomenovana konstanta, kedze by bolo fajn ju spomenut aj v texte vo template. Ale kam ju zavesit, aby bola dostupna aj v template?

Reply:
:ok: Pridal som ju do `context` v `render_delete_form`

Reply:
@viliambalaz Ok, len pls pomenuj tu konstantu `ADMIN_EXTEND_SNOOZE_BY_DAYS`, nech je jasne, ze to je extend adminom, nie userom. Kedze user extenduje o 3 dni. Casom mozno aj tych userovych 3 dni vysunieme do konstanty. A analogicky uprav aj komentar pri konstante.

Reply:
:ok: